SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -507,6 +507,19 @@ void USpatialSender::SendInterestBucketComponentChange(const Worker_EntityId Ent
 	}
 }
 
+void USpatialSender::SendTearOffUpdate(Worker_EntityId EntityId, Worker_ComponentId ComponentId, bool bTornOff)
```

Comment:
`bTornOff` is always true. Is it ever useful to send an update setting tear off to 0?